sklearn/multioutput.py

```diff
@@ -233,6 +233,18 @@ class MultiOutputRegressor(RegressorMixin, _MultiOutputEstimator):
     ----------
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.datasets import make_regression
+    >>> from sklearn.multioutput import MultiOutputRegressor
+    >>> from sklearn.linear_model import LinearRegression
+    >>> X, y = make_regression(n_features=4, random_state=0)
```

Comment:
You can use `sklearn.datasets.load_linnerud` instead. This will be multivariate directly.

Reply:
Good to know! Done.